CHANGELOG.md

```diff
@@ -521,10 +521,6 @@ decorators, in favor of a simpler inheritance-based approach to defining models
 * react `16.13 -> 17.0`
 * semver `added @ 7.3`
 
-#### 📚 Required App Library Upgrades
-
-* ag-Grid `23.x -> 24.1`
-
```

Comment:
This was for an older version - we didn't use this format in any subsequent release notes, so cleaning up from here for consistency.